### PR TITLE
fixing tabs and ensuring columns appear

### PR DESCRIPTION
**Pull request by alisaraa**
This is a change from v.1.1.0 but I can only merge into master. Plz help me. 
1) renaming Schema to fields
2) renaming Comments to Table Definition
3) making sample invisible
4) ensuring that the right variables are passed to the schema hbs

### CODE REVIEW

**Comment by alisaraa** [comment]
@jaeger401  I have a merge conflict because I'm trying to update v.1.1.0 but I can only make a PR to master...plz help. This github is not v.1.1.0 then I guess.

File: wherehows-web/app/templates/datasets/dataset.hbs [hunk 1]
```diff
@@ -191,11 +191,13 @@
         {{/tablist.tab}}
       {{/if}}
 
-      {{#unless isSFDC}}
-        {{#tablist.tab tabIds.SampleData on-select=(action "tabSelectionChanged")}}
-          Sample Data{{/tablist.tab}}
-      {{/unless}}
-
+      {{!--feature not available--}}
```
Comment (jaeger401) [inline wherehows-web/app/templates/datasets/dataset.hbs]: Is there a reason not to strip this out entirely?

File: wherehows-web/app/templates/datasets/dataset.hbs [hunk 2]
```diff
@@ -169,9 +169,9 @@
         {{#tablist.tab tabIds.Access on-select=(action "tabSelectionChanged")}}ACL Access{{/tablist.tab}}
       </span>
 
-      {{#tablist.tab tabIds.Comments on-select=(action "tabSelectionChanged")}}Comments{{/tablist.tab}}
+      {{#tablist.tab tabIds.Comments on-select=(action "tabSelectionChanged")}}Table Definition{{/tablist.tab}}
```
Comment (jaeger401) [inline wherehows-web/app/templates/datasets/dataset.hbs]: Do we no longer need the `Comments` tab? Or is the reason for cannibalizing it here that it's smaller surgery to achieve the task at hand? Would it be too big a lift to rename the tab identifier as well?

A similar question goes for the `Schema` tab below.

Comment (alisaraa) [reply wherehows-web/app/templates/datasets/dataset.hbs]: I'm renaming comments to table def-- it just looks like I killed it and added another.
Comments for them is...comments on this entry. Which is...comments on this table...which is definition. I discussed with @GordonWongEZ  and he wants to make it clear from the get go that we are putting definitions not general comments. We had an issue at Fitbit where people kind of ignored the table comments.

**Comment by alisaraa** [comment]
@brafkind  from this branch, I can see the columns in the JSON tab, just not the column tab. This is probably because it's not parsing the json within model.schema correctly (making it a string). I'll make it a backlog ticket. I know it thinks its a string because it gives its length as 1k+. Ugh.

**Comment by brafkind** [comment]
See #9 for fixing the table display of fields

**Comment by alisaraa** [comment]
@jaeger401  do you have any more outstanding questions on this one?

File: wherehows-web/app/templates/datasets/dataset.hbs (hunk 2 above)
Comment (jaeger401) [reply wherehows-web/app/templates/datasets/dataset.hbs]: OK, cool. Sorry if my question wasn't clear -- I was curious about changing the display string without changing the corresponding `tabIds.Comments` to be `tabIds.TableDefinitions`.

**Comment by alisaraa** [comment]
For what one?

File: wherehows-web/app/templates/datasets/dataset.hbs (hunk 2 above)
Comment (alisaraa) [reply wherehows-web/app/templates/datasets/dataset.hbs]: Oh my mistake, I misunderstood. That’d be fine but note that it’s called comments throughly the code— dataset-comments is it’s hbs file. I thought just changing it in the front end prevented a weird arbitrary shift from comments to table definitions within the code base, which is ultimately unavoidable so let me know what you think about the best way to handle it

**Pull request merged**